core: allow deleting of evaluations (#13492)
hashicorp/nomad: website/content/api-docs/evaluations.mdx

}
```

## Delete Evaluations

This endpoint deletes evaluations. In order to utilise this endpoint the
eval broker should be paused via the
[update_scheduler_configuration][operator scheduler update configuration] API
endpoint.

This API endpoint should be used cautiously and only in outage situations where
there is a large backlog of evaluations not being processed. During most normal

> and outage scenarios, Nomads reconciliation and state management will handle

Reviewer: ```suggestion
and outage scenarios, Nomad's reconciliation and state management will handle
```

evaluations as needed.

| Method    | Path              | Produces           |
| --------- | ----------------- | ------------------ |
| `DELETE`  | `/v1/evaluations` | `application/json` |

The table below shows this endpoint's support for
[blocking queries](/api-docs#blocking-queries) and
[required ACLs](/api-docs#acls).

| Blocking Queries | ACL Required |
| ---------------- | ------------ |
| `NO`             | `management` |

### Parameters

> - `:EvalIDs` `(array<string>: <required>)`- An array of evaluation UUIDs to

Reviewer 2: ```suggestion
- `EvalIDs` `(array<string>: <required>)`- An array of evaluation UUIDs to
```

>   delete. This must be the full UUID, not the short 8-character one.

Reviewer: ```suggestion
  delete. This must be a full length UUID and not a prefix.
```
